app/modules/ads/tracking/slot-tracker.js

```diff
@@ -1,5 +1,9 @@
 const trackingRouteName = 'special/adengadinfo';
 
+function getPosParameter({pos = ''}) {
+	return (Array.isArray(pos) ? pos : pos.split(',')).shift();
```

Comment:
`shift` modifies original array. Can we get first element without changing reference?

Reply:
You are right! thanks!